app/views/calls/_new_call.html_BACKUP_7141.erb

```diff
@@ -0,0 +1,31 @@
+  <div class="modal fade new-call" tabindex="-1" role="dialog" id="call-<%= p.patient.primary_phone %>">
```

Comment:
@dfelsent This file looks like an artifact from a text editor crash -- can you remove it? Thanks much!